scipy/stats/stats.py

```diff
@@ -5815,12 +5815,13 @@ def ttest_ind(a, b, axis=0, equal_var=True, nan_policy='propagate',
         The 'omit' option is not currently available for permutation tests or
         one-sided asympyotic tests.
 
-    permutations : int or None (default), optional
-        The number of random permutations that will be used to estimate
-        p-values using a permutation test. If `permutations` equals or exceeds
-        the number of distinct permutations, an exact test is performed
-        instead (i.e. each distinct permutation is used exactly once).
+    permutations : non-negative int, np.inf, or None (default), optional
         If None (default), use the t-distribution to calculate p-values.
```

Comment:
```suggestion
        If 0 or None (default), use the t-distribution to calculate p-values.
```

Reply:
I agree with `0` indicating to use the t-distribution rather than a permutation test here.